include/ozo/connection.h

```diff
@@ -433,7 +433,11 @@ inline decltype(auto) get_socket(T&& conn) noexcept {
 template <typename T>
 inline decltype(auto) get_io_context(T& conn) noexcept {
     static_assert(Connection<T>, "T must be a Connection");
+#if BOOST_VERSION < 106700
     return get_socket(conn).get_io_context();
+#else
+    return get_socket(conn).get_executor().context();
+#endif
```

Comment:
Looks like this condition is not needed, because in Boost 1.66  `boost::asioposix::stream_descriptor::get_executor()` [already exists](https://www.boost.org/doc/libs/1_70_0/doc/html/boost_asio/reference/posix__basic_stream_descriptor/get_executor.html), so it can be used safely.